### PR TITLE
Gene table related doc update

### PR DESCRIPTION
**Pull request by yichaoS**
- Remove gencode step (human genes only) from "update gene & gene alias" doc
- Remove PDB step from "import seed database" doc

### CODE REVIEW

**Review by n1zea144 — APPROVED** [review]
Approved, but I would wait for Hyve approval too.

**Review by sheridancbio — APPROVED** [review]
👍  good cleanup

**Comment by sheridancbio** [comment]
@inodb @SRodenburg : before this gets merged, we will wait for your approval / change requests. Or if you do not plan to review, just let us know.

File: docs/Updating-gene-and-gene_alias-tables.md
Comment (SRodenburg) [inline docs/Updating-gene-and-gene_alias-tables.md:68]: Some different example values for mouse

```suggestion
./importGenes.pl --gtf <gencode.vM25.annotation.gtf> --genome-build <GRCm38>
```

**Review by SRodenburg — COMMENTED** [review]
Looks good, thanks for the cleanup

**Pull request merged**
